tests/python/contrib/test_clml/menangerie.py

```diff
@@ -0,0 +1,1527 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Use the models in python/tvm/relay/testing if possible

Reply:
Yes we can use existing networks from testing instead additional network file. I have removed the file and used network from testing.